content/log/index.md

```diff
@@ -0,0 +1,135 @@
+log_dd_url: intake.logs.datadoghq.com
+# The port to use when submitting the log lines
+log_dd_port: 10515
+{{< /highlight >}}
+
+## Collect logs with an Integrations
+To start collecting logs for an integration, you need to add a log sections in your integration yaml file, this log section need to contain:
+
+* `type` : type of log input source (**tcp** / **udp** / **file**)
+* `port` / `path` : mandatory option to specify depending on `type`. For **tcp**/**udp** `type` you have to specify the `port`, for **file** you need to specify its `path`
+* `service` : name of the service owning the log
+* `source` : \/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
+* `sourcecategory` : \/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
+
+### Examples
+#### File log collection configuration
+If you want to gather you nginx logs for instance stored in **/var/log/access.log** and **/var/log/error.log** you would edit your `nginx.yaml` file as follow
+
+{{< highlight yaml >}}
+init_config:
+instances:
+(...)
+#Log section
+logs:
+  - type: file
+    path: /var/log/access.log
+    appname: nginx
+    source: nginx
+    sourcecategory: http_access
+    tags: env:prod 
+
+  - type: file
+    path: /var/log/error.log
+    appname: nginx
+    source: nginx
+    sourcecategory: http_error
+{{< /highlight >}}
+
+#### TCP/UDP Log collection configuration
+
+If your java application doesn't log into a file but forwards its logs on the local 10514 port in TCP edit your `jmx.yaml` file as such in order to listen to this port and forward those logs to your datadog application.:
+
+{{< highlight yaml >}}
+init_config:
+instances:
+(...)
+#Log section
+logs:
+    - type: tcp
+      port: 10514
+      appname: java
+      source: java
+
+{{< /highlight >}}
+
+## Collect log from a custom log file
+
+To send custom log file that would not be part of an existing integration. Rename `custom-logs.yaml.example` in `custom-logs.yaml` and edit the desired parameters as seen in the previous section.
+
+{{< highlight yaml >}}
+logs:
+  - type: file
+    path: /path/to/your/file.log
+    appname: name_of_your_app
+    tags: my_tag_name:my_tag_value
+    source: source_name
+{{< /highlight >}}
+
+## Reserved attributes 
+
+This section is relevant only if you log are JSON formated.
+If so, you need to be aware that some JSON attributes have a specific interaction with your datadog application:
+
+### `date` attribute
+By default your datadog application generates a timestamp for your lag that corresponds to the reception date. This field is used to display your data over the timeline.
+
+But if your log contain a date reserved attribute, then its value is considered as the official log timestamp, date reserved attributes are:
+
+* `timestamp`
+* `date`
+* `_timestamp`
+* `Timestamp`
+* `eventTime`
+* `published_date`
+
+Define any attribute as the official timestamp of your logs with the [log date remapper processor](/log/processing/#log-date-remapper)
```

Comment:
"Define any attribute as the official timestamp of your logs with the"
CHANGE TO

"You can also ask Datadog to consider any attribute as the official date reference thanks to the"

Reply:
👍